scripts/system/create/modules/entityShapeVisualizer.js

```diff
@@ -35,14 +36,6 @@ function getEntityShapePropertiesForType(properties) {
                     modelURL: properties.compoundShapeURL,
                     localDimensions: properties.localDimensions
                 };
-            } else if (properties.shapeType === "sphere") {
```

Comment:
we can remove this because non-uniform sphere zones are no longer a special case